vxmessenger/transport.py

```diff
@@ -476,48 +616,14 @@ def handle_outbound_message(self, message):
         self.log.info("MessengerTransport outbound %r" % (message,))
         reply = self.construct_reply(message)
         self.log.info("Reply: %s" % (reply,))
-        try:
-            resp = yield self.request(
-                method='POST',
-                url='%s?access_token=%s' % (self.config['outbound_url'],
-                                            self.config['access_token']),
-                data=json.dumps(reply, separators=(',', ':')),
-                headers={
-                    'Content-Type': 'application/json',
-                },
-                pool=self.pool)
-
-            data = yield resp.json()
-            self.log.info('API reply: %s' % (data,))
-
-            if resp.code == http.OK:
-                yield self.publish_ack(
-                    user_message_id=message['message_id'],
-                    sent_message_id=data['message_id'])
-                yield self.add_status(
-                    component='outbound',
-                    status='ok',
-                    type='request_success',
-                    message='Request successful')
-            else:
-                yield self.nack(
-                    message, data['error']['message'],
-                    self.SEND_FAIL_TYPES.get(
-                        data['error']['code'], 'request_fail_unknown'))
-        except (TimeoutError,), e:
-            yield self.nack(message, e, 'request_fail_unknown')
 
-    @inlineCallbacks
-    def nack(self, message, reason, status_type):
-        yield self.publish_nack(
-            user_message_id=message['message_id'],
-            sent_message_id=message['message_id'],
-            reason=reason)
-        yield self.add_status(
-            component='outbound',
-            status='down',
-            type=status_type,
-            message=reason)
+        request = {
+            'message_id': message['message_id'],
+            'method': 'POST',
+            'relative_url': 'v2.6/me/messages',
```

Comment:
Also, please extract this as the url from the `outbound_url` since the API has broken before between 2.5 & 2.6, so can likely happen in future again.